andela/activo-api
# #159102652 Remove Content-Type Validation From GET Requests

#### What does this PR do?
>- Removes Content-Type validation from GET requests
#### Description of Task to be completed?
>- Remove Content-Type validation from GET requests
#### How should this be manually tested?
>- Run `git fetch origin bg-remove-content-159102652:bg-remove-content-159102652`, then run `git checkout bg-remove-content-159102652`
>- Run `flask run`
>- Send a GET request to `localhost:5000/api/v1/asset-categories` without the `Content-Type` header
>- You should successfully receive the list of asset categories
#### Any background context you want to provide?
>- Currently, sending a GET request to endpoints without adding the `Content-Type: application/json` header returns the error:
```
{
    "message": "Content-Type should be application/json",
    "status": "error"
}
```
which is not ideal for GET requests. This PR fixes this problem.
#### What are the relevant pivotal tracker stories?
>- [#159102652](https://www.pivotaltracker.com/story/show/159102652)
#### Screenshots (if appropriate)
>- N/A
#### Questions:
>- N/A
